### DISCUSSION TITLE
Got 'Uncaught TypeError' during installation

### DISCUSSION THREAD

**Discussion by rakkasumi** [wavelog/wavelog#646] [Q&A] [answered]
I'm trying to setup wavelog via docker and got this: 
`Fatal error: Uncaught TypeError: Cannot access offset of type string on string in /var/www/html/install/index.php:101 Stack trace: #0 {main} thrown in /var/www/html/install/index.php on line 101`

The `docker-compose.yaml` as same as wiki's, only changed MARIADB_PASSWORD.

**Comment by HB9HIL** [comment]
> I'm trying to setup wavelog via docker and got this: `Fatal error: Uncaught TypeError: Cannot access offset of type string on string in /var/www/html/install/index.php:101 Stack trace: #0 {main} thrown in /var/www/html/install/index.php on line 101`
> 
> The `docker-compose.yaml` as same as wiki's, only changed MARIADB_PASSWORD.

this issue is known and already fixed in `dev` branch.  Workaround is to set your browser language to english.

**Reply by rakkasumi** [reply]
thanks for reply, but the dev branch image doesn't seem to be updated yet, is there a plan to update it when it will? Or could you please rerun the build action manually?

**Reply by HB9HIL** [reply, marked as answer]
Allright. The `dev` images was just recreated.

Please try again. Use `ghcr.io/wavelog/wavelog:dev` as image in `wavelog-main` in your docker-compose.yml

**Reply by rakkasumi** [reply]
works perfect now
